src/html.sortable.js

```diff
@@ -685,6 +692,9 @@ var sortable = function (sortableElements, options) {
       if (!dragging || !_listsConnected(sortableElement, dragging.parentElement)) {
         return
       }
+      if (maxItems && _filter(_getChildren(sortableElement), _data(sortableElement, 'items')).length >= maxItems) {
```

Comment:
I get your point: https://github.com/lukasoppermann/html5sortable/pull/257 (you talking about "class" confused me, I think it means something else outside of js). The clean solution would be to add a new function which filters children differently to only include items with a sortable-item-id.

Until we have this working, can you please add the `-1` thingy and a comment `//TODO: remove -1 once placeholder items are not included when getting children`.

Otherwise people will have a bug.